docs/okeps/okep-5259-no-overlay.md

```diff
@@ -0,0 +1,1491 @@
+
+The pod-to-remote-pod egress traffic will enter the host VRF via the
+ovn-k8s-mpX interface. Then it will be forwarded to remote nodes according to
+the host routing table of the VRF.
+
+An example traffic path is as follows (assuming pod IP of 10.128.0.10,
+destined to remote pod 10.128.2.11):
+
+The routing table of the default VRF of node A which contains learnt BGP routes:
+
+```text
+10.128.0.0/16 via 10.128.2.1 dev ovn-k8s-mp0
+10.128.0.0/24 dev ovn-k8s-mp0 proto kernel scope link src 10.128.0.2
+10.128.1.0/24 nhid 30 via 172.18.0.2 dev eth0 proto bgp metric 20
+10.128.2.0/24 nhid 25 via 172.18.0.4 dev eth0 proto bgp metric 20
+```
+
+```mermaid
+sequenceDiagram
+    participant Pod1 on Node A
+    participant ovn_cluster_router Node A
+    participant Host VRF/Kernel Node A
+    participant Underlay Physical Network
+    participant Node B
+
+    Pod1 on Node A->>ovn_cluster_router Node A: Packet to remote Pod2 (10.128.2.11)
+    ovn_cluster_router Node A->>Host VRF/Kernel Node A: Forward packet to host via ovn-k8s-mp0
+    Note over Host VRF/Kernel Node A: BGP route lookup
+    Host VRF/Kernel Node A->>Host VRF/Kernel Node A: Match BGP route in host table: 10.128.2.0/24 via 172.18.0.3 (Node B's IP)
+    Host VRF/Kernel Node A->>Underlay Physical Network: Route packet out via breth0
+    Underlay Physical Network->>Node B: Packet arrives at Node B
+    Note over Node B: Traffic is delivered to Pod2
+```
+
+And the reply path:
+
+```mermaid
+sequenceDiagram
+    participant Node B
+    participant Underlay Physical Network
+    participant Host VRF/Kernel Node A
+    participant ovn_cluster_router Node A
+    participant Pod1 on Node A
+    
+    Note over Node B: Traffic is delivered from Pod2
+    Node B->>Underlay Physical Network: Packet leaves Node B
+    Underlay Physical Network->>Host VRF/Kernel Node A: Route packet in via breth0
+    Note over Host VRF/Kernel Node A: route lookup
+    Host VRF/Kernel Node A->>Host VRF/Kernel Node A: Match static route: 10.128.0.0/24 via 100.64.0.1 (Node A's pod subnet)
+    Host VRF/Kernel Node A ->>ovn_cluster_router Node A: forwarded via mp0
+    Note over ovn_cluster_router Node A: route lookup
+    ovn_cluster_router Node A->>ovn_cluster_router Node A: Match the direct route to local pods
+    ovn_cluster_router Node A->>Pod1 on Node A: Packet to local Pod1 (10.128.0.5) via the node switch
+```
+
+#### Import Routes to NodeSubnets
+
+Changes to BGP behavior are necessary to import node subnet routes into the
+gateway router and the host routing table of each node. A new `transport` key
+will be included in the OVN CNI NetConf (the NAD's spec.config JSON) to indicate
+the east-west transport protocol of the network. If a network operates in
+no‑overlay mode, this key is set to `noOverlay`. This information can then be
+passed into the NetInfo object and utilized by both the RouteAdvertisement
+controller and the route import controller of ovnkube-controller.
```

Comment:
There is no RouteAdvertisement controller in ovnkube-controller, I think it is ok if you just mention the route import controller here